data/json/items/gun/44.json

```diff
@@ -20,11 +20,7 @@
     "durability": 7,
     "valid_mod_locations": [ [ "accessories", 2 ], [ "barrel", 1 ], [ "bore", 1 ], [ "grip", 1 ], [ "mechanism", 4 ], [ "muzzle", 1 ], [ "rail", 1 ], [ "sights", 1 ], [ "stock", 1 ], [ "underbarrel", 1 ] ],
     "magazine_well": 1,
-    "magazines": [
-      [ "44", [ "deaglemag" ] ],
-      [ "45", [ "m1911mag", "m1911bigmag" ] ],
-      [ "9mm", [ "glockmag", "glockbigmag" ] ]
-    ]
+    "magazines": [ [ "44", [ "deaglemag" ] ], [ "45", [ "m1911mag", "m1911bigmag" ] ], [ "9mm", [ "glockmag", "glockbigmag" ] ] ]
```

Comment:
This should wrap at each ammo type. Forcing it all on one line hurts readability due to `[]` spam.

Reply:
Hmm, I think the real problem here is that this is an array of arrays of arrays in the first place, it would wrap nicely if it were:
```
"magazines" : {
  "44" : ["deaglemag"],
  "45": [ "m1911mag", "m1911bigmag" ]
}
```

Reply:
That would require it not to be a proper, well defined object though.
It would be very non-standard to have it use data as member names.

Reply:
A json object is just a key/value map, we just happen to have a lot of objects that map directly to game objects, but it's in no way nonstandard to have a data/data map.